Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -161,7 +161,8 @@ bool GameFile::TryLoadVolume()
 
   m_game_id = QString::fromStdString(volume->GetGameID());
   std::string maker_id = volume->GetMakerID();
-  volume->GetTitleID(&m_title_id);
+  if (std::optional<u64> title_id = volume->GetTitleID())
+    m_title_id = *title_id;
```

Comment:
It looks like the current UI code does not really care about GetTitleID failing and using an uninitialised/zero u64 for the title ID. With a std::optional<u64>, more explicit checks would have to be added though, so I think this would be better left to a future PR.